crates/typos-cli/src/bin/typos-cli/main.rs

```diff
@@ -216,6 +216,12 @@ fn run_checks(args: &args::Args) -> proc_exit::ExitResult {
             let overrides = overrides
                 .build()
                 .with_code(proc_exit::sysexits::CONFIG_ERR)?;
+            if walk_policy.force_exclude() && path.is_file() {
```

Comment:
Why the file check?

Reply:
Good question... I think I over-focused on my specific use case, sorry.